src/container/container.ts

```diff
@@ -260,7 +278,8 @@ class Container implements interfaces.Container {
           this._bindingDictionary.clone(),
           this._middleware,
           this._activations.clone(),
-          this._deactivations.clone()
+          this._deactivations.clone(),
+          this._moduleActivationStore.clone()
```

Comment:
I could not think of a test quickly for the cloning of the moduleActivationStore.

Reply:
A cloned container should call the handler, so we can check a deactivation handler is called once a cloned container receives an unbind request. That could be a way to test it

Reply:
What I was looking for is a test that would fail if I did not clone.  I don't think there is one.  The store for the module handlers is just for removing from deactivations and activations.  Cloning ensures we do not hold on to a handler from a module added after snapshot when we restore.  If we did not clone there would not be a bug just inefficient code.
I will just add a test that the snapshot gets a clone.